packages/services/src/competition.service.ts

```diff
@@ -2788,64 +2754,38 @@ export class CompetitionService {
         params.pagingParams,
       );
 
-      // If user is authenticated, enrich competitions with voting information
+      // If user is authenticated, enrich competitions with trading constraint information
```

Comment:
I think this comment is wrong?  This isn't gated on the user being authenticated anymore

Reply:
Fixed.